shell.nix

```diff
@@ -18,6 +18,8 @@ in
 mkShell {
   name = "extensions-shell";
   buildInputs = [
+    autoflake
```

Comment:
what's autoflake and isort?

Reply:
They format the import sequence on the .py files.

Reply:
Correction. It removes unused imports.